apps/carenet-api/requirements.txt

```diff
@@ -3,4 +3,6 @@ fastapi==0.108.0
 pydantic==2.5.3
 pinecone-client==2.2.4
 sentence-transformers==2.2.2
-python-dotenv==1.0.0
\ No newline at end of file
+python-dotenv==1.0.0
+beautifulsoup4==4.12.3
```

Comment:
the api not using bs4 , the crawler feeds , please add there the requirements.txt

Reply:
Done @liranzxc